cmd/template/cluster/flag.go

```diff
@@ -300,7 +296,7 @@ func (f *flag) Validate() error {
 	}
 
 	if f.Name != "" {
-		valid, err := key.ValidateName(f.Name, f.EnableLongNames)
+		valid, err := key.ValidateName(f.Name, true)
```

Comment:
Do we ever call `ValidateName` with `false`? Otherwise we might not need the argument at all.

Reply:
We call it from within the GenerateName method, and pass `enableLongNames` parameter from it.